codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/AuthSchemeInterceptorSpec.java

```diff
@@ -148,20 +149,17 @@ private MethodSpec generateAuthSchemeParams() {
         if (!authSchemeSpecUtils.useEndpointBasedAuthProvider()) {
             builder.addStatement("$T operation = executionAttributes.getAttribute($T.OPERATION_NAME)", String.class,
                                  SdkExecutionAttribute.class);
+            builder.addStatement("$T.Builder builder = $T.builder().operation(operation)",
```

Comment:
This part is needed because earlier Builder was built in a single line
Since we have conditional bases builder for sigv4a so splited this generation code
```
QueryAuthSchemeParams.builder().operation(operation).region(region).build();
```